references/segmentation/v2_extras.py

```diff
@@ -20,7 +20,7 @@ def _transform(self, inpt, params):
         if not params["needs_padding"]:
             return inpt
 
-        fill = self.fill[type(inpt)]
+        fill = v2._geometry._get_fill(self.fill, type(inpt))
```

Comment:
Nit: this is where it is defined. `_geometry` just imports it

```suggestion
        fill = v2._utils._get_fill(self.fill, type(inpt))
```